googlecode_wiki_to_github_markdown/googlecode_wiki_to_github_markdown.py

```diff
@@ -0,0 +1,159 @@
+import argparse
+import re
+import os
+
+# Used to remove internal wiki links cancellation :
+# https://code.google.com/p/support/wiki/WikiSyntax#Internal_wiki_links
+project_name = "GwtQuery"
+
+
+def convert_to_md(wiki_file_name):
+    with open(wiki_file_name) as f:
+        wiki = f.read()
+
+    wiki = remove_gplusone(wiki)
+    wiki = remove_toc(wiki)
+    wiki = remove_labels(wiki)
+    wiki = remove_internal_link_cancellations(wiki)
+
+    wiki = convert_internal_links(wiki)
+    wiki = convert_http_links(wiki)
+    wiki = convert_numbered_lists(wiki)
+    wiki = convert_headers(wiki)
+    wiki = convert_code_snippets_markers(wiki)
+    wiki = convert_comments(wiki)
+
+    wiki = remove_extra_spaces(wiki)
+    wiki = remove_extra_empty_lines(wiki)
+
+    return wiki
+
+
+def remove_gplusone(wiki):
+    return re.sub(r"<g:plusone.*</g:plusone>", "", wiki)
+
+
+def remove_toc(wiki):
+    return re.sub(r"<wiki:toc.*/>", "", wiki)
+
+
+def remove_labels(wiki):
+    return re.sub(r"#labels.*", "", wiki)
+
+
+def remove_internal_link_cancellations(wiki):
+    return wiki.replace("!" + project_name, project_name)
```

Comment:
It didn't really matter since you probably don't have a "!GQuery" string in your project, but I agree it doesn't make sense to have this string hardcoded at the top of the file. Updated!